src/openms/source/ANALYSIS/ID/BayesianProteinInferenceAlgorithm.cpp

```diff
@@ -70,13 +69,18 @@ namespace OpenMS
         cnt_(0)
     {}
 
-    unsigned long operator() (IDBoostGraph::Graph& fg) {
+    unsigned long operator() (IDBoostGraph::Graph& fg, unsigned int idx) {
       //TODO do quick bruteforce calculation if the cc is really small?
-      cnt_++;
+
       // this skips CCs with just peps or prots. We only add edges between different types.
       // and if there were no edges, it would not be a CC.
       if (boost::num_vertices(fg) >= 2)
       {
+        OPENMS_LOG_DEBUG << "Running cc " << String(idx) << "..." << std::endl;
```

Comment:
this gets called very often (potentially), right?
Don't use `std::endl` then... awfully slow...

Reply:
Thanks. I was actually thinking about ways to avoid logging completely when multithreading. What do you think?